application-services/custom/camera-management/appcamera/app.go

```diff
@@ -58,14 +58,9 @@ func (app *CameraManagementApp) Run() error {
 
 	devices, err := app.getAllDevices()
 	if err != nil {
-		app.lc.Errorf("Unable to query all devices")
-
-	} else if devices == nil {
-		app.lc.Debugf("No devices found")
+		app.lc.Warnf("no devices found: %s", err.Error())
```

Comment:
i think this still is an error retrieving devices, and not that no devices were found